### ISSUE TITLE
Getting started questions

### ISSUE THREAD

**Issue by aejolene** [matrix-msu/kora-connect#3]
I'm getting started and I have some questions.
1. Is there an order in which I should be doing things?
2. ~~I'm not sure I understand the library as opposed to the galleries. Could you explain?~~ Thanks @watrall!
3. When adding items to a gallery or to the library, what does the "Fields" field do? Is this what users will see? Or is that controlled in another way from the library?
4. ~~After I created my gallery, clicking on the individual objects threw Page Not Found error. Do I need to manually make pages for each object? Link to my test gallery is [here](http://vaarchaeology.matrix.msu.edu/index.php/reports/).~~ Moved this to another issue
5. ~~As I look for themes, is there anything to keep in mind for best compatibility with the way KORA works? I'm new to WordPress, but not to html/etc.~~
6. How do I control the style of KORA objects as they display? For example, I'd like to edit how the thumbnail scales and change the title text appearance on the WP pages.
7. Is there a way to automate adding objects to the front end as I ingest them to the back end repository?

Thanks in advance.

**Comment by watrall** [comment]
think about a library as similar to the WordPress media library.  Basically represents the collection of all the objects you've pulled in from your KORA repository.  A gallery is a public facing thing.  You build your gallery and then insert it into a page or a post.  As for #4, could you open a separate issue (that looks like an actual, real bug).  As for themes, there is nothing optimized for the plugin (we've thought about building one).

**Comment by aejolene** [comment]
Updated the initial comment, striking through answered/moved questions and asking a couple more.

**Comment by watrall** [comment]
1. I'm not sure. @seilagonzalez?  
2. With this version of the plugin, there is unfortunately very little control over things like this.  We are planning on implementing more styling options in future version of the plugin
3. I'm pretty sure that you need to manually add things once they've been added to KORA.  This is an interesting idea, however.

**Comment by seilagonzalez** [comment]
I don't think there is a predefine order.

**Comment by seilagonzalez** [comment]
Fields? where in gallery or library? could you be more specific?...

**Comment by seilagonzalez** [comment]
YOu cannot control the style.

**Comment by seilagonzalez** [comment]
You can add object into the frontend (add new kora object) without having to touch library or gallery.

**Comment by aejolene** [comment]
![image](https://cloud.githubusercontent.com/assets/13810938/17623164/b356cff8-606c-11e6-8669-26562387daeb.png)
This is what I mean by fields. It's in Object, Library, and Gallery.

**Comment by asheill** [comment]
1. You have to start in "Connect" to identify your kora installation, schemes you're searching over, and the search tokens in KORA. From there I'd do "Add New KORA Object" to start adding records to your "Library" and then you can start making groups of images with "Galleries" From there, you want to go into "Pages" and put your Galleries and Objects on your website.

**Comment by asheill** [comment]
@aejolene "Field(s)" are the metadata fields that you want to display on the short display of the record. It's what you will see on a browse page before selecting the object to see the full record. 
![screenshot 2016-08-12 09 33 24](https://cloud.githubusercontent.com/assets/20991107/17623793/db245444-606f-11e6-88c7-bfe454391c1a.png)

**Comment by aejolene** [comment]
@asheill Perfect. Thanks. That's why I was getting "undefined" outputs, then. It's all coming together...

**Comment by asheill** [comment]
@aejolene a "Gallery" is a group of objects. A "Gallery" can be displayed on a "Page". An individual item with your "Library" can also be displayed on a "Page".

**Comment by asheill** [comment]
@seilagonzalez regarding: "You can add object into the frontend (add new kora object) without having to touch library or gallery." can you specify if the object will then be inserted into your Library?

**Comment by seilagonzalez** [comment]
@asheill I had that wrong. I just talk to @watrall about it. Apparenly, everything that you include on a page or post, and on a gallery should be into the Library. I will put somebody to fix that.

**Comment by aejolene** [comment]
New question (that may be a bug, in which case I'll move): is there something I need to do to get search to work? Related, I'd like to have an "advanced search" screen, querying on multiple fields with picklist choices. I'm confident that I could write the queries to do that, but it is it possible in this case? I'm new to PHP and not sure how it works.

**Comment by asheill** [comment]
@aejolene I noticed this morning that the search definitely doesn't seem to be buggy when you "Add New Object to Kora" and I reported it to programming. I just tried now and it seems to be working. Definitely buggy. I believe the concept is that it should advanced search over the Field(s) you've selected to display. If you don't limit your search at all before clicking the "Search" button it should return all available records from KORA.
